packages/@uppy/core/src/Uppy.ts

```diff
@@ -308,7 +308,7 @@ const defaultUploadState = {
  * Manages plugins, state updates, acts as an event bus,
  * adds/removes files and metadata.
  */
-export class Uppy<M extends Meta, B extends Body> {
+export class Uppy<M extends Meta, B extends Body = Record<string, never>> {
```

Comment:
Don't we need to set it on other plugins as well? Should we maybe export it as `EmptyBody` or `DefaultBody` for readability?

Reply:
We don't have to, this is the only place people have to define it and only on core will you be able to infer it. As in, the only way to actually run into something with the `Body` generic is using core methods.